appservice/src/deploy/deploy.ts

```diff
@@ -75,6 +75,7 @@ export async function deploy(client: SiteClient, fsPath: string, context: IDeplo
         ext.outputChannel.appendLog(localize('deployStart', 'Starting deployment...'), { resourceName: client.fullName });
         try {
             if (!context.deployMethod && config.scmType === ScmType.GitHub) {
+                context.valuesToMask.push(client.fullName);
```

Comment:
This should be tackled on a more generic basis - we want to mask `client.fullName` in as many cases as possible. Here are a few examples of where I've added "more generic" fixes:
https://github.com/microsoft/vscode-azuretools/blob/main/ui/src/registerCommand.ts#L34
https://github.com/microsoft/vscode-azuretools/blob/main/ui/src/tree/AzExtTreeDataProvider.ts#L177